### PR TITLE
send_telegram: add support for markdown template

### PR DESCRIPTION
**Pull request by tsnoam**
per the discussion in http://discuss.flexget.com/t/new-plugin-telegram-cli-telegram-bot/1655/26?u=tsnoam

### CODE REVIEW

File: flexget/plugins/output/send_telegram.py [hunk 1]
```diff
@@ -67,6 +69,9 @@ class SendTelegram(object):
     `template`::
     Optional. The template from the example is the default.
 
+    `use-markup`::
```
Comment (gazpachoking) [inline flexget/plugins/output/send_telegram.py]: Up or down?

Comment (tsnoam) [inline flexget/plugins/output/send_telegram.py]: oops :)
fixing in a jiffy.

**Comment by gazpachoking** [comment]
Thanks! Want to update the wiki with this option?

**Comment by tsnoam** [comment]
sure. done.

**Pull request merged**